WORKSPACE

```diff
@@ -430,11 +433,25 @@ hazel_repositories(
             hazel_hackage("shake", "0.17.8", "ade4162f7540f044f0446981120800076712d1f98d30c5b5344c0f7828ec49a2") +
             hazel_hackage("filepattern", "0.1.1", "f7fc5bdcfef0d43a793a3c64e7c0fd3b1d35eea97a37f0e69d6612ab255c9b4b") +
             hazel_hackage("terminal-progress-bar", "0.4.0.1", "c5a9720fcbcd9d83f9551e431ee3975c61d7da6432aa687aef0c0e04e59ae277") +
+            hazel_hackage("rope-utf16-splay" , "0.2.0.0", "83d1961bf55355da49a6b55d6f58d02483eff1f8e6df53f4dccdab1ac49e101d") +
```

Comment:
Have you asked on the rope bug tracker for this to be added to Stackage? My experience is most authors are happy to do so, and it reduces the number of lines here (as an example, I did for terminal-progress-bar, so that line above disappears on the next Stackage role).